images/plbase/plbase-install.sh

```diff
@@ -103,6 +103,7 @@ uv venv --seed
 # don't exist on Amazon Linux 2023. We need to override them with the correct paths.
 export CC=/usr/bin/gcc
 export CXX=/usr/bin/g++
+export LDSHARED="gcc -shared"
```

Comment:
You may want to match the above with `/usr/bin/gcc -shared`

Reply:
Fixed in [896fced](https://github.com/PrairieLearn/PrairieLearn/pull/11039/commits/896fced5f4ffbfe287d92523ff51af856d145bed).